### ISSUE TITLE
Make app mobile-friendly

### ISSUE THREAD

**Issue by CarlosNZ** [openmsupply/conforma-web-app#1548]

**Comment by regotaina** [comment]
The margin on the right side looks slightly tight for me.

Does anyone else has the same?
![Screen Shot 2023-11-06 at 1 06 20 PM](https://github.com/openmsupply/conforma-web-app/assets/132425096/a904a6f3-dc1a-4047-982e-fb6281d62b5d)

It happens on a few forms, like 'Vincular Utilizador' :
![Screen Shot 2023-11-06 at 2 19 24 PM](https://github.com/openmsupply/conforma-web-app/assets/132425096/544c213b-41b9-4c64-9c4a-7a5b8f29af99)

**Comment by CarlosNZ** [comment]
> The margin on the right side looks slightly tight for me. Maybe part of [#1558](https://github.com/openmsupply/conforma-web-app/issues/1558)

Yeah, we haven't really done anything on the review home page yet.

**Comment by guatemartin** [comment]
#1600 has hopefully addressed the review side of things.
